### PR TITLE
Ship shim with integration testing 

### PR DESCRIPTION
**Pull request by pakrym**
And cleanup test projects.

### CODE REVIEW

File: src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/IISApplication.cs [hunk 1]
```diff
@@ -327,6 +328,7 @@ private string CreateTestSiteName()
 
         private string GetAncmLocation(string dllRoot)
         {
+            dllRoot = Environment.CurrentDirectory;
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/IISApplication.cs]: ??

Comment (pakrym) [reply src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/IISApplication.cs]: That's why it's WIP.

File: test/TestTasks/TestTasks.csproj [hunk 2]
```diff
@@ -1,8 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.2;net461</TargetFrameworks>
```
[the remark this answers is not in the document]
Comment (pakrym) [reply test/TestTasks/TestTasks.csproj]: No reason.

**Comment by pakrym** [comment]
>Make sure this can be consumed cross repo. Other than that, things look good.


Did it in https://github.com/aspnet/MetaPackages/pull/281

**Pull request merged**